tests/contrib/aioredis/test_aioredis.py

```diff
@@ -0,0 +1,156 @@
+import aioredis
+import pytest
+
+from ddtrace import Pin
+from ddtrace.contrib.aioredis.patch import aioredis_version
+from ddtrace.contrib.aioredis.patch import patch
+from ddtrace.contrib.aioredis.patch import unpatch
+from ddtrace.vendor.wrapt import ObjectProxy
+from tests.utils import override_config
+
+from ..config import REDIS_CONFIG
+
+
+@pytest.fixture
+async def redis_client():
+    r = await get_redis_instance()
+    yield r
+
+
+def get_redis_instance():
+    if aioredis_version >= (2, 0):
+        return aioredis.from_url("redis://localhost:%s" % REDIS_CONFIG["port"])
+    return aioredis.create_redis(("localhost", 6379))
+
+
+@pytest.fixture(autouse=True)
+async def traced_aioredis(redis_client):
+    await redis_client.flushall()
+
+    patch()
+    try:
+        yield
+    finally:
+        unpatch()
+    await redis_client.flushall()
+
+
+def test_patching():
+    """
+    When patching aioredis library
+        We wrap the correct methods
+    When unpatching aioredis library
+        We unwrap the correct methods
+    """
+    if aioredis_version >= (2, 0):
+        assert isinstance(aioredis.client.Redis.execute_command, ObjectProxy)
+        assert isinstance(aioredis.client.Redis.pipeline, ObjectProxy)
+        assert isinstance(aioredis.client.Pipeline.pipeline, ObjectProxy)
+        unpatch()
+        assert not isinstance(aioredis.client.Redis.execute_command, ObjectProxy)
+        assert not isinstance(aioredis.client.Redis.pipeline, ObjectProxy)
+        assert not isinstance(aioredis.client.Pipeline.pipeline, ObjectProxy)
+    else:
+        assert isinstance(aioredis.Redis.execute, ObjectProxy)
+        unpatch()
+        assert not isinstance(aioredis.Redis.execute, ObjectProxy)
+
+
+@pytest.mark.asyncio
+@pytest.mark.snapshot
+@pytest.mark.skipif(aioredis_version < (2, 0), reason="")
+async def test_basic_request_13(redis_client):
+    val = await redis_client.get("cheese")
+    assert val is None
+
+
+@pytest.mark.skipif(aioredis_version >= (2, 0), reason="")
+async def test_basic_request_2(redis_client):
```

Comment:
We're not snapshotting this test case?

Reply:
we don't need to mark this as `pytest.mark.asyncio`

Reply:
I removed `mark.asyncio` from all the tests in this file